### PR TITLE
Expand list of solang code owners

### PR DESCRIPTION
**Pull request by seanyoung**
When creating a pull request, the users in this files are used to populate the reviewers. If it lists @hyperledger/solang-committers then we can't see who those users are and as soon as one of the members of @hyperledger/solang-committers leaves a review, then the pull request is marked as approved while we really want *all* the maintainers to approve.

Signed-off-by: Sean Young <sean@mess.org>

### CODE REVIEW

**Comment by seanyoung** [comment]
I've added @ryjones but left @hyperledger/solang-committers  in place as the reviewers to demonstrate the problem.

**Review by xermicus — APPROVED** [review]

**Comment by xermicus** [comment]
We have now this exact situation, e.g. need to remember to manually request a review from Lucas

**Pull request merged**
